lib/cgi/cookie.rb

```diff
@@ -123,6 +132,14 @@ def secure=(val)
       @secure
     end
 
+    # Set whether the Cookie is a httponly cookie or not.
+    #
+    # +val+ must be a boolean.
+    def httponly=(val)
+      @httponly = val if val == true or val == false
```

Comment:
Yes, I understand that.  I'd choose `@secure = !!val` or simply `attr_accessor :secure` and just leave the truthiness to Ruby.  Let's see how the maintainer feels.